OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/NativeFindCoursesFragment.java

```diff
@@ -116,10 +123,13 @@ public void onDestroy() {
     public static class ViewHolder {
         public final ListView listView;
         public final View loadingIndicator;
+        public final TextView no_courses;
 
         public ViewHolder(View view) {
             this.listView = (ListView) view.findViewById(R.id.course_list);
             this.loadingIndicator = view.findViewById(R.id.loading_indicator);
+            this.no_courses = (TextView) view.findViewById(R.id.no_courses);
+
```

Comment:
This extra line can be removed.

Reply:
If i remove that line the class will throw an error, not sure exactly what you mean here

Reply:
I actually meant the extra carriage-return that was added @ line # 192.

Reply:
This is still pending.